src/Capture.php

```diff
@@ -89,13 +89,20 @@ class Capture
      */
     protected $userAgentString = '';
 
+	/**
```

Comment:
@MASNathan can you remove this "Nitpick-CI" bot, its spamming my dashboard like crazy